src/data/actions/courseEnrollments.js

```diff
@@ -28,6 +35,10 @@ const fetchCourseEnrollments = (enterpriseId, options) => (
         dispatch(fetchCourseEnrollmentsSuccess(response.data));
       })
       .catch((error) => {
+        if (error.response.status === 404) {
+          dispatch(fetchCourseEnrollmentsSuccess(emptyEnrollments));
```

Comment:
I could see someone coming back to this code and being confused why we're doing this - might want to add a comment explaining that the API returns a 404 if there are no enrollments found and this prevents the page from treating that as an error?